server/dash/design.py

```diff
@@ -47,6 +47,24 @@
 }
 
 
+# TODO: Fixing multiple streetlights and waterwaste color
+# Need to add another discrete color
+DISCRETE_COLORS_MAP = {
```

Comment:
Just double checking, did you get these from https://github.com/hackforla/311-data/blob/b0717ad25eab4ef25a2f26da187d9d8905ab5f77/server/api/alembic/seeds/types.csv? If so, could you leave a comment to that effect?

Reply:
Added link to PR description

Reply:
Oops sorry, I meant to leave a comment in the code. That way, when someone reads the code, they can cross-reference it with types.csv.